tools/block-generator/run_postgres.sh

```diff
@@ -28,7 +28,7 @@ function start_postgres() {
      -e POSTGRES_PASSWORD=algorand \
      -e PGPASSWORD=algorand \
      -p $POSTGRES_PORT:5432 \
-     postgres
+     postgres:13-alpine
```

Comment:
I had to use a version with md5 password encryption for it to work with golang's pg driver.

Reply:
woah, is that how you get it to work? You can also put `sslmode=disable` in the connection string.

Reply:
sslmode=disable was included but it was returning error `pq: unknown authentication response: 10` when the generator queries the db.